src/conf.c

```diff
@@ -74,8 +74,10 @@ int conf_loadfile( conf_t *conf, char *file )
 		line ++;
 
 		*s = 0;
-		fscanf( fp, "%100[^\n#]s", s );
-		fscanf( fp, "%*[^\n]s" );
+		if( fscanf( fp, "%100[^\n#]s", s ) == 0 )
+			fprintf( stderr, _("Error in %s line %i.\n"), file, line );
+		if( fscanf( fp, "%*[^\n]s" ) == 0 )
```

Comment:
The effect of the second call to `fscanf` as well as the `fgetc` could be merged into the first call to `fscanf`. Something like:

	fscanf( fp, "%100[^\n#]s%*[^\n]s%*[\n]s" )

Also, in this case, a return value of `EOF` isn't being handled.

Reply:
I don't think you can merge those calls: if the first pattern is not matched (i.e. line starting with a comment), not matching for the second pattern will be attempted, thus preventing the file pointer from proceeding forward. I believe this is the reason why those calls are split on 3 different lines.